CHANGELOG.md

```diff
@@ -37,7 +37,25 @@ Bottom level categories:
 - Hal
 -->
 
-## Unreleased
+## 22.0.0 (2024-07-17)
+
+### Overview
+
+### Our first major version release!
+
+For the first time ever, WGPU is being released with a major version (i.e., 22.* instead of 0.22.*)! Maintainership has decided to fully adhere to [Semantic Versioning](https://semver.org)'s recommendations for versioning production software. According to [SemVer 2.0.0's own Q&A about when to use 1.0.0 and beyond](https://semver.org/spec/v2.0.0.html#how-do-i-know-when-to-release-100):
+
+> ### How do I know when to release 1.0.0?
+> 
+> If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backward compatibility, you should probably already be 1.0.0.
+
+It is a well-known fact that WGPU has been used for applications and platforms already in production for years, at this point. We are often concerned with tracking breaking changes, and affecting these consumers' ability to ship. By releasing our first major version, we publicly acknowledge that this is the case. We encourage other projects in the Rust ecosystem to follow suit.
+
+### Overview
+
+A major ([pun intended](#our-first-major-version-release)) theme of this release is incremental improvement. Beyond the typically large set of bug fixes, new features, and other adjustments to WGPU, @wumpf and @teoxoy have merged many simplifications to WGPU's internals. Many of these change WGPU to use atomically reference-counted resource tracking (i.e., `Arc<…>`), rather than using IDs to manage the lifetimes of platform-specific graphics resources of separate reference counts. This has led us to diagnose and fix many long-standing bugs, and net some neat performance improvements on the order of 30% of some worklaods.
+
+While the above is exciting, we acknowledge already finding and fixing some (easy-to-fix) regressions from the above work. If you migrate to WGPU 22 and encounter such bugs, please engage us in the issue tracker right away!
 
```

Comment:
suggestion: RE: some chatter in [Matrix](https://matrix.to/#/!FZyQrssSlHEZqrYcOb:matrix.org/$6F_MRK8BwNA_KRdqURJjYJBFn0XGpOy1Xg7PE8Afe14?via=matrix.org&via=mozilla.org&via=envs.net) about expectations with "stable" versioning: We should probably also add a disclaimer paragraph at the end stating that we do not anticipate that the rate of breaking changes will slow down for the foreseeable future.

CC @kpreid.

Reply:
Added this:

> Note that while we start to use the major version number, WGPU is _not_ "going stable", as many Rust projects do. We anticipate many breaking changes before we fully comply with the WebGPU spec., which we expect to take a small number of years.